<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research</title>

<script src="site_libs/header-attrs-2.21/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/lumen.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.2.1/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.2.1/css/v4-shims.min.css" rel="stylesheet" />
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-119400399-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-119400399-1');
</script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Home</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="papers.html">Papers</a>
</li>
<li>
  <a href="software.html">Software</a>
</li>
<li>
  <a href="teaching.html">Teaching</a>
</li>
<li>
  <a href="http://juliawrobel.com/tutorials/tutorials.html">Tutorials</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:julia.wrobel@cuanschutz.edu">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="Downloads/wrobel_cv.pdf">
    <span class="ai ai-cv ai-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://scholar.google.com/citations?user=QI8GrFUAAAAJ&amp;hl=en&amp;oi=ao">
    <span class="ai ai-google-scholar-square ai-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/julia-wrobel/">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Research</h1>

</div>


<p><link rel="stylesheet" href="academicons.css"/>
<link rel="stylesheet" href="styles.css" type="text/css"></p>
<div id="functional-data-analysis" class="section level2">
<h2>Functional Data Analysis</h2>
<p>My functional data work is broadly focused on methods for functional
registration, regression, and dimension reduction. A subset of this work
is summarized below.</p>
<div id="curve-registration-for-functional-data" class="section level3">
<h3>Curve Registration for Functional Data</h3>
<p>I developed a method to <strong>register exponential family
functional data</strong>, which was published in <a
href="https://onlinelibrary.wiley.com/doi/abs/10.1111/biom.12963">Biometrics</a>
in 2019. Our methods are implemented in the <code>registr</code>
package, an <code>R</code> package on CRAN and GitHub you can download
<a href="https://github.com/julia-wrobel/registr">here</a>.</p>
<p>If each curve represents an observation for one subject, then
<em>curve registration</em> refers to warping the domain (often time) of
a set of curves so that the main features of each curve are aligned
across subjects. An example with simulated data is below.</p>
<p><br><img src="images/registration.png" alt="registration" style="width:75%"><br></p>
<p><em>Warping functions</em> (center) are applied to the
<em>unregistered curves</em> (left) to get <em>registered</em> curves
(right). The <em>exponential family</em> part comes in because not all
functional observations are Gaussian or continuous. Our approach to
registration allows alignment of data that is discrete as well as
continuous.</p>
<p>Below our method is applied to accelerometer data where each
subject’s binary activity (active vs. not active) is collected at every
minute over 24 hours. Periods of inactivity are colored in light blue
and periods of activity are colored in dark blue. Applying our
registration technique to the activity data pulls out patterns in
physical activity.</p>
<p><br><img src="images/binaryRegistration.png" alt="binary registration" style="width:75%"><br></p>
<ol style="list-style-type: decimal">
<li><strong>J Wrobel</strong> (2018). <a
href="https://joss.theoj.org/papers/9c40c4f0ede1827cc5a9430c625d6494">registr:
Registration for exponential family functional data</a>. <em>The Journal
of Open Source Software</em>.</li>
<li><strong>J Wrobel</strong>, V Zipunnikov, J Schrack, and J Goldsmith
(2019). <a
href="https://onlinelibrary.wiley.com/doi/full/10.1111/biom.12963">Registration
for exponential family functional data</a>. <em>Biometrics</em>.</li>
<li><strong>J Wrobel</strong> and A Bauer (2021). <a
href="https://joss.theoj.org/papers/10.21105/joss.02964.pdf">registr
2.0: Incomplete Curve Registration for Exponential Family Functional
Data</a>. <em>The Journal of Open Source Software</em>.</li>
<li>E McDonnell, V Zipunnikov, J Schrack, J Goldsmith, and <strong>J
Wrobel</strong> (2022). <a
href="https://www.tandfonline.com/doi/abs/10.1080/09291016.2021.1929673">Registration
of 24-hour accelerometric rest-activity profiles and its application to
human chronotypes</a>. <em>Biological Rhythm Research</em>.</li>
</ol>
</div>
</div>
<div id="single-cell-multiplex-imaging" class="section level2">
<h2>Single Cell Multiplex Imaging</h2>
<p>Since 2019 I have been developing tools and methodology for analysis
of <strong>multiplex single cell imaging</strong> (MI) data, an emerging
image analysis technique that has revolutionized researchers’ ability to
study tissue structure and function at a cellular level while preserving
the original spatial context. <em>Single cell</em> refers to individual
cell resolution, <em>multiplex</em> refers to multiple types of proteins
in the tissue that are tagged, allowing for identification of nuanced
cell subtypes, function, and tissue regions, and <em>imaging</em>
indicates that biological spatial relationships in the tissue are
preserved. An example image from a <a
href="https://www.sciencedirect.com/science/article/abs/pii/S1556086421021754">paper</a>
I worked on studying tumor-immune relationships in non-small cell lung
carcinoma is below.</p>
<p><br><img src="images/multiplexImage.png" alt="registration" style="width:75%"><br></p>
<p>This image is a multichannel TIFF files, where each channel
represents signal intensity of a particular protein. The left panel is
the composite image combining all 8 channels collected for this
dataset.The next three images are three individual channels from the
same image, shown, from left to right, the nucleus (DAPI) channel, the
tumor (cytokeratin) channel, and the immune (CD8) channel.</p>
<p>Multiplex imaging data has a complex data acquisition, image
processing, and analysis pipeline with unique challenges that can be
addressed by statisticians. First, tissue is placed on a slide and
labeled with multiple (multiplex) antibody markers. These images are
then segmented to identify tissue compartments (e.g. tumor vs. stroma),
cells, and nuclei. Marker intensities across samples must be normalized
and batch corrected to account for non-biological variability. Then
cells are <em>phenotyped</em>, or given biological label(s) based on
their mean marker intensities. Finally, the datasets are undergo
compositional and spatial analysis, potentially in combination with
patient level outcomes such as survival time, disease subtype, or cancer
stage. To address these challenges, I have developed methods and
software for normalizing multiplex imaging data (1,2), for analyzing
continuous marker expression using density-based variation analysis (3),
for spatial analysis using scalar spatial summaries (4), and for spatial
analysis using techniques drawn from functional data analysis (5,
6).</p>
<ol style="list-style-type: decimal">
<li>CR Harris, <strong>J Wrobel</strong>, SN Vandekar. <a
href="https://joss.theoj.org/papers/10.21105/joss.04180.pdf">mxnorm: An
R Package to Normalize Multiplexed Imaging Data (2022)</a>. <em>The
Journal of Open Source Software</em>.</li>
<li>CR Harris, ET McKinley, JT Roland, Q Liu, MJ Shrubsole, KS Lau, RJ
Coffey, <strong>J Wrobel</strong>, and SN Vandekar (2022). <a
href="https://pubmed.ncbi.nlm.nih.gov/34983062/">Quantifying and
correcting slide-to-slide variation in multiplexed immunofluorescence
images.</a> <i> Bioinformatics</i>.</li>
<li>S Seal, T Vu, T Ghosh, <strong>J Wrobel</strong>, and D Ghosh
(2022). <a
href="https://academic.oup.com/bioinformaticsadvances/article/2/1/vbac039/6590640">DenVar:
Density-based Variation analysis of multiplex imaging data</a>.
<em>Bioinformatics Advances</em>.</li>
<li>B Steinhart, KR Jordan, J Bapat, MD Post, LW Brubaker, BG Bitler,
and <strong>J Wrobel</strong> (2021). <a
href="https://pubmed.ncbi.nlm.nih.gov/34615692/">The Spatial Context of
Tumor-Infiltrating Immune Cells Associates with Improved Ovarian Cancer
Survival</a>. <em>Molecular Cancer Research</em>.</li>
<li>T Vu, <strong>J Wrobel</strong>, BG Bitler, EL Schenk, KR Jordan,
and D Ghosh (2022). <a
href="https://pubmed.ncbi.nlm.nih.gov/35704658/">SPF: a spatial and
functional data analytic approach to cell imaging data</a>. <em>PLoS
Computational Biology</em>.</li>
<li>T Vu, S Seal, <strong>J Wrobel</strong>, T Ghosh, M Ahmadian, D
Ghosh (2022). <a
href="https://www.biorxiv.org/content/10.1101/2022.06.17.496475v1.full.pdf">FunSpace:
A functional and spatial analytic approach to cell imaging data using
entropy measures</a>. <em>bioRxiv</em>.</li>
</ol>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
